static/app/components/events/autofix/autofixRootCause.tsx

```diff
@@ -187,7 +207,16 @@ function SuggestedFixSnippet({
       >
         {snippet.snippet}
       </StyledCodeSnippet>
-    </div>
+      {sourceLink && (
+        <CodeLinkWrapper>
```

Comment:
Could you add a frontend test assertion that ensures this link shows up when it should?